src/main/resources/db/changelog/changes/07-add-users.yaml

```diff
@@ -11,7 +11,7 @@ databaseChangeLog:
                   value: "harry.potter@gmail.com"
               - column:
                   name: password
-                  value: "$2a$12$NDd3ta.qEAHYOrQ9m3iJ9u..FSIeVkYuXqATEgXLvbhrDYwRgNApa"
+                  value: "12341234"
```

Comment:
let's not change the scripts that were already merged to main branch, better to create a new ones and try to fix the situation there instead